website/events/models/registration_information_field.py

```diff
@@ -40,6 +40,15 @@ class RegistrationInformationField(models.Model):
         _("required"),
     )
 
+    delete_after = models.DateField(
```

Comment:
Why don't we delete this after a predetermined time? This way it could technically be saved forever if people enter a value that is years in the future.

Reply:
Also possible. The reason I opened the issue myself is because we don't have an exact policy on this, and regarding data minimization I figure that some kinds of data might have different retention times than others and this forces organizers to think about it. We could add a validator of `max 3 years` or something. But I'm open for discussion on this

Reply:
I would personally say that 1/2-ish months after the activity ended would also be fine. So many options. As long as it is not 'unlimited'.

Reply:
As a constant? Or as max value?

Reply:
Constant

Reply:
@pingiun what do you think?

Reply:
Then I think 2 months after the event is over we can delete the registration info

Reply:
Isn't it a bit our job to have thought about this for them?

Reply:
No, because we don't know what kind of data this is, so we are not able to make a judgement on what the retention period should be.